docs/tutorial/tutorial-3.rst

```diff
@@ -386,59 +386,115 @@ or doing other pre-distribution tasks.
+        If you're on a RHEL-based distribution, you'll see:
 
-      [helloworld] Building .rpm package...
-      Generating rpmbuild layout... done
+        .. code-block:: console
 
-      Write RPM spec file... done
+          (beeware-venv) $ briefcase package
 
-      Building source archive... done
+          [helloworld] Finalizing application configuration...
+          Targeting fedora:36 (Vendor base rhel)
+          Determining glibc version... done
 
-      Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.Kav9H7
-      + umask 022
-      ...
-      + exit 0
-      Building RPM package... done
+          Targeting glibc 2.35
+          Targeting Python3.10
+
+          [helloworld] Building .rpm package...
+          Generating rpmbuild layout... done
+
+          Write RPM spec file... done
+
+          Building source archive... done
+
+          Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.Kav9H7
+          + umask 022
+          ...
+          + exit 0
+          Building RPM package... done
+
+          [helloworld] Packaged dist/helloworld-0.0.1-1.fc36.x86_64.rpm
+
+        The ``dist`` folder will contain the ``.rpm`` file that was generated.
+
+      .. group-tab:: Arch Based
+
+        If you're on an Arch-based distribution, you'll see:
+
+        .. code-block:: console
+          
+          (beeware-venv) $ briefcase package
+
+          [helloworld] Finalizing application configuration...
+          Targeting manjaro:rolling (Vendor base arch)
+          Determining glibc version... done
+          Targeting glibc 2.37
+          Targeting Python3.10
+
+          [helloworld] Building .pkg.tar.zst package...
+          Generating pkgbuild layout... done
+          Building source archive... done
+          Write PKGBUILD file... done
+          ==> Making package: helloworld 0.0.1-1 (Sat 26 Aug 2023 06:21:16 AM EDT)
+          ==> Checking runtime dependencies...
+          ==> Checking buildtime dependencies...
+          ==> Retrieving sources...
+            -> Found helloworld-0.0.1.tar.gz
+          ==> Validating source files with md5sums...
+              helloworld-0.0.1.tar.gz ... Skipped
+          ==> Extracting sources...
+            -> Extracting helloworld-0.0.1.tar.gz with bsdtar
+          ==> Entering fakeroot environment...
+          ==> Starting package()...
+          ==> Tidying install...
+            -> Removing libtool files...
+            -> Purging unwanted files...
+            -> Removing static library files...
+            -> Compressing man and info pages...
+          ==> Checking for packaging issues...
+          ==> Creating package "helloworld"...
+            -> Generating .PKGINFO file...
+            -> Generating .BUILDINFO file...
+            -> Adding changelog file...
+            -> Generating .MTREE file...
+            -> Compressing package...
+          ==> Leaving fakeroot environment.
+          ==> Finished making: helloworld 0.0.1-1 (Sat 26 Aug 2023 06:21:17 AM EDT)
```

Comment:
We don't need to include every line of output - we can redact the minor details.
```suggestion
          ...
```